fetch.bs

```diff
@@ -1538,6 +1601,27 @@ for each associated <a for="fetch group">fetch record</a> whose
     <var>origin</var>. [[!HTTP]] [[!HTTP-SEMANTICS]] [[!HTTP-COND]] [[!HTTP-CACHING]] [[!HTTP-AUTH]]
     [[!TLS]]
 
+    <p>If <var>credentials</var> is true and the user agent supports <a for=/>Token Binding</a>,
+    send <a for=/>token-binding protocol version</a> and
+    <a for=/>token-binding key parameters</a> in a
+    <code>token_binding</code> Client Hello Extension, as described in
+    <a href="https://tools.ietf.org/html/draft-ietf-tokbind-negotiation#section-2">section 2</a>
+    of the Token Binding Negotiation spec [[!TOKBIND-NEGOTIATION]].
+    If Token Binding negotiation succeeds, set
+    <a for=/>token-binding protocol version</a> and
+    <a for=/>token-binding key parameters</a>
+    in <var>connection</var>'s <a for=connection>TLS connection metadata</a>
+    to the negotiated values,
+    and <a for=/>token-binding Exported Keying Material</a> to the computed value.
+
+    <p class="note no-backref">
+    The user agent will use <a for=/>Token Binding</a> for any
+    <a for=/>request</a> sent over a TLS connection for which Token Binding
+    negotiation was successful.
+    <a for=/>Token Binding</a> is only proposed when <var>credentials</var>
+    is true, so connections which successfully negotiate a binding will never be
+    pooled with connections that do not include credentials.
```

Comment:
A UA would have a different key for `google.com` and `youtube.com` (to use that example). Since Token Binding messages are per-request, it's fine for those origins to be coalesced on the same H2 connections - different Token Binding messages using different keys will be sent for the requests on the different origins. So, there are different bindings for `google.com` and `youtube.com`.

The part where things get tricky with coalescing is that the use of Token Binding for requests on that connection is negotiated once for the connection. The practical effect of this is that all origins that can transitively be coalesced should support the same Token Binding protocol version and key parameter (and anything else that might get negotiated in a future version of Token Binding).

Reply:
I added clarifying language.

@mikewest , please let me know if it needs more tweaking.